test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
It was in order as the last item, because "-h" comes last, since all the others have two dashes. "--" always comes before "-<letter>". If you put the descriptions into a JavaScript array and call .sort(), ideally it should match the order in the file.